cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -96,6 +96,29 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 	if err != nil {
 		return err
 	}
+
+	fmt.Println("Build stages results:")
+
+	// Include the build stages output inside the worker's logs.
+	for _, stage := range osbuildJobResult.OSBuildOutput.Build.Stages {
```

Comment:
@achilleas-k This doesn't work for v2 manifests, does it?

Reply:
The worker only depends on osbuild1 though. Is a bunch of osbuild2 output ignored now when returning the jobresult, and just what's overlapped in json tags between v1 and v2 results?

Reply:
osbuild2 output gets unmarshalled into an osbuild1.Result object automatically.  @ondrejbudai remember this? :)

https://github.com/osbuild/osbuild-composer/blob/1abd9a11924cf20a2a3b9e2cbe173dae4b614005/internal/osbuild1/result.go#L53

Reply:
Sidenote: At some point (soon) I think we should change this to use v2 types primarily at which point we would need to change occurrences like this as well.

Reply:
Ah, I remember now, we need to rewrite that at some point but this PR is definitely a good step forward, thanks! :)